docs/monitoring.md

```diff
@@ -74,8 +74,8 @@ follows:
   <tr>
     <td>spark.history.provider</td>
     <td>org.apache.spark.deploy.history.FsHistoryProvider</td>
-    <td>Name of the class implementing the application history backend. Currently there is only
-    one implementation, provided by Spark, which looks for application logs stored in the
+    <td>Name of the class implementing the application history backend. The default implementation,
+     the FsHistoryProvider, which looks for application logs stored in the
```

Comment:
Grammar sounds weird; perhaps "The default implementation is FsHistoryProvider, which..."?